apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx

```diff
@@ -354,6 +367,30 @@ export const ApplicationCertificateWrapper: FunctionComponent<ApplicationWrapper
                                 readOnly={ readOnly }
                                 data-testid={ `${testId}-certificate-type-radio-group` }
                             />
+
+                            {
+                                showCertificateRemovalWarning && (
+                                    <Message
+                                        data-componentid="application-certificate-removal-warning-message"
```

Comment:
Any reason to add both data-componentid and data-test id?

Reply:
Resolved in. https://github.com/wso2/identity-apps/pull/3462/commits/d131f1bd945f4ce8f681eb32dd7fb1261c0c5930